vault/token_store.go

```diff
@@ -1128,7 +1128,8 @@ func (ts *TokenStore) handleTidy(req *logical.Request, data *framework.FieldData
 
 		for _, child := range children {
```

Comment:
Please add some progress logging so people can be aware things are still happening.